src/available-command.ts

```diff
@@ -602,6 +602,13 @@ export function possibleTechTiles(engine: Engine, player: Player) {
       data: { tiles }
     });
   }
+
+  commands.push({
+    name: Command.DeclineTechTile,
```

Comment:
It can be `Command.Decline`, with `data: { offer: 'up'}` or `data: { offer: 'up-${field}}` when only one field is upgradable. The Decline command is already used for leech / itars declining to spend power in their gaia phase, it can also be used for this.

If you're worried about the viewer, we can just make the viewer add the command `${Command.Decline} ${data.offer}` that way 'decline 2pw' or 'decline up-nav' would show in the logs.